app/routes/news._index.mdx

```diff
@@ -26,6 +26,25 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'July 30, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Circulars are Now Available via Kafka, New Schema Release v4.1.0 and Heartbeat Kafka Topic</Anchor>
```

Comment:
- Reorder so that the order of this list agrees with the news banner.
- Always use the Oxford comma.

```suggestion
        <Anchor>Circulars are Now Available via Kafka, Heartbeat Kafka Topic, and New Schema Release v4.1.0</Anchor>
```